tcms/management/management/commands/baseurl.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
```

Comment:
Let's move all of these files under `tcms/core`. The `tcms.management` module has a different meaning and is a legacy name.